regression/contracts/assigns-enforce-malloc-zero/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--enforce-contract foo _ --malloc-may-fail
+^EXIT=0$
+^SIGNAL=0$
+^\[foo\.\d+\] line 22 Check that a\[\(signed long int\)i\] is assignable: SUCCESS
```

Comment:
Use `signed (long (long )?)?int` to ensure this works across several platforms.

Reply:
fixed